app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -9,6 +9,7 @@ COPYRIGHT="2007-2017 David Given"
 LICENSE="MIT"
 REVISION="1"
 SOURCE_URI="https://github.com/davidgiven/wordgrinder/archive/$portVersion.tar.gz"
+PATCHES=wordgrinder-$portVersion.patchset
```

Comment:
cosmetic change, to comply with the [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering):
```diff
-PATCHES=wordgrinder-$portVersion.patchset
 CHECKSUM_SHA256="54085af7963e1f67342bc0b1b20d1ccc75494f2e23d401b601cf3089acea747c"
+PATCHES="wordgrinder-$portVersion.patchset"
```